src/cmd_say.rs

```diff
@@ -0,0 +1,144 @@
+use anyhow::Result;
+use clap::Parser;
+
+/// Prints your text in a text bubble with KittyCAD as ASCII art
+///
+///     $ kittycad say
+///     $ kittycad say hello!
+///     $ kittycad say Hello World!
+#[derive(Parser, Debug, Clone)]
+#[clap(verbatim_doc_comment)]
+pub struct CmdSay {
+    /// What kitty says
+    #[clap(name = "input", required = false, multiple_values = true)]
+    pub input: Vec<String>,
+}
+
+#[async_trait::async_trait]
+impl crate::cmd::Command for CmdSay {
+    async fn run(&self, ctx: &mut crate::context::Context) -> Result<()> {
+        let kitty_speaking = self.input.len() > 0;
+        let kitty_string = format_kitty(kitty_speaking);
+        if self.input.len() > 0 {
```

Comment:
We can reuse `kitty_speaking` here. And above we should use `!self.input.is_empty()` like clippy suggests.